recipe/meta.yaml

```diff
@@ -23,11 +25,12 @@ requirements:
   - bob.core
   - bob.io.base
   - giflib
-  - libpng
-  - jpeg
-  - libtiff
+  - libpng 1.6*
+  - jpeg 9*
+  - libtiff 4.0.*
   - pkg-config
-  - gcc     # [linux]
+  - isl 0.12.2  # [linux]
```

Comment:
We should probably add a comment along those lines (pointing back to https://github.com/conda-forge/staged-recipes/issues/436#issuecomment-212323816)